content/TrueCommand/SystemManagement/_index.md

```diff
@@ -18,7 +18,12 @@ See [Alert Management]({{< relref "/TrueCommand/Alerts/AlertManage.md" >}}) for
 
 ![DashboardSingleSystemView](/images/TrueCommand/2.0/DashboardSingleSystemView.png "Dashboard Single System View")
 
-There are several "hot spots" on the card that will open system specific areas for management. 
+
+*Storage* lists how many pools and drives are being used by the system. It also shows how much storage is used and available by size and percentage.
+
+*ARC MISS* shows how often the system is going to disk rather than using the ARC cache. Anything above 0% means that the system is using ram. The numbers may vary depending on use case and work load.
```

Comment:
```suggestion
*ARC MISS* shows how often the system is using disks instead of the ARC cache. Anything above 0% means that the system is using RAM. The numbers vary on use case and work load.
```